scripts/create-partner-chart

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
So in general I'm very hesitant to add anything to the scripts stuff, but I suppose since this is actually just a partner charts folder layout generator thats used one time it should be ok.

Reply:
If we are coming up with something new soon, I would definitely prefer waiting on that before we ask partners to add charts here by themselves and just migrate them for them. We are asking them to use the rancher/helm3-charts repo now, which means their charts need to be Helm 3 compatibl and that should make migrating them as easy as populating the package.yaml, add partner annotation, and copy overlays.

Reply:
I think package.yaml and annotations should stay roughly the same in future, we are just rebuilding framework that builds them out. But if we make a decision around adding or removing annotation or change to package.yaml format, we are going to have to have that done on all partner charts.